### PR TITLE
Feat/play

### PR DESCRIPTION
**Pull request by szk2s**
Implementation of MPE playback feature

### CODE REVIEW

**Comment by szk2s** [comment]
I separated my commits on this branch from clean-up-codes #20 completely.

File: lib/play.js [hunk 1]
```diff
@@ -1,11 +1,40 @@
...
+    resolve();
+  });
+};
+
+const outputPorts = () => {
+  return JZZ().info().outputs;
+};
+
+const refreshPorts = () => {
+  JZZ.refresh();
```
Comment (zigen) [inline lib/play.js]: you can write like `const refreshPorts = JZZ.refresh;`

File: lib/play.js [hunk 2]
```diff
@@ -1,11 +1,40 @@
...
-        let player = data.smf.player();
-        player.connect(midiout);
-        player.play();
-        resolve(data);
+const JZZ = require('jzz');
+require('jzz-midi-smf')(JZZ);
+const utils = require('./utils');
+
+const smfPlay = (smf, portIdx) => {
+  return new Promise((resolve) => {
```
Comment (zigen) [inline lib/play.js]: you can remove a bracket and `return` since this function returns Promise directly.

**Comment by szk2s** [comment]
Thanks @zigen , I fixed as you said.

**Pull request merged**